ingestion/src/metadata/ingestion/source/pipeline/airflow/metadata.py

```diff
@@ -152,6 +156,26 @@ def _extract_serialized_task(task: Dict) -> Dict:
             return task["__var"]
         return task
 
+    def get_all_tags(self, dag_id: str) -> List[str]:
+        tag_query = (
+            self.session.query(DagTag.name)
+            .filter(DagTag.dag_id == dag_id)
+            .distinct()
+            .all()
+        )
+        return [tag[0] for tag in tag_query]
+
```

Comment:
lets add some exception handling here, in case the query fails to run, in the scenario of user does not have permission to access this table, the entire pipeline ingestion won't fail!

Reply:
Done